tests/pmap_test.py

```diff
@@ -877,6 +877,36 @@ def f(x):
     expected = grad(lambda x: np.sum(f(x)))(x)
     self.assertAllClose(ans, expected, check_dtypes=False)
 
+  def testPmapStaticArgnums(self):
+    @partial(pmap, axis_name='i', static_argnums=1,
+             devices=xla_bridge.devices())
```

Comment:
The default is to use all devices, so you can remove the `devices` argument (I probably added them to other tests but I can't remember why now).